app/graphql/mutations/create_covid_run_mutation.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Mutations
+  # Mutation to create a COVID run.
+  class CreateCovidRunMutation < BaseMutation
+    argument :flowcells, [Types::Inputs::Ont::FlowcellInputType], required: true
+
+    field :run, Types::Outputs::Ont::RunType, null: true
+    field :errors, [String], null: false
+
+    def resolve(flowcells:)
```

Comment:
What does `flowcells:` do - with the colon?

Reply:
When you define a method and you put a colon after an argument name it becomes a named argument.  When this method is called by the GraphQL library, it expects to be able to pass the fields as named arguments with the same name as the field.